Avoid intermittent test failures with manual_seed()

My [Travis tests failed](https://travis-ci.org/pytorch/audio/jobs/630111792?utm_medium=notification&utm_source=github_status) for reasons ostensibly not related to [my code change](https://github.com/pytorch/audio/pull/378):
```
    def test_griffinlim(self):
        tensor = torch.rand((1, 1000))
    
        n_fft = 400
        ws = 400
        hop = 100
        window = torch.hann_window(ws)
        normalize = False
        momentum = 0.99
        n_iter = 8
        length = 1000
        rand_init = False
        init = 'random' if rand_init else None
    
        specgram = F.spectrogram(tensor, 0, window, n_fft, hop, ws, 2, normalize).sqrt()
        ta_out = F.griffinlim(specgram, window, n_fft, hop, ws, 1, normalize,
                              n_iter, momentum, length, rand_init)
        lr_out = librosa.griffinlim(specgram.squeeze(0).numpy(), n_iter=n_iter, hop_length=hop,
                                    momentum=momentum, init=init, length=length)
        lr_out = torch.from_numpy(lr_out).unsqueeze(0)
    
>       self.assertTrue(torch.allclose(ta_out, lr_out, atol=5e-5))
E       AssertionError: False is not true
test/test_functional.py:94: AssertionError
```


Adding calls to `manual_seed()` in `test_functional.py` should make these tests deterministic, no?

--- a/test/test_functional.py
+++ b/test/test_functional.py
@@ -71,6 +71,10 @@ class TestFunctional(unittest.TestCase):
 
     @unittest.skipIf(not IMPORT_LIBROSA, 'Librosa not available')
     def test_griffinlim(self):
+
+        # NOTE: This test is flaky without a fixed random seed
+        # See https://github.com/pytorch/audio/issues/382
+        torch.random.manual_seed(42)
         tensor = torch.rand((1, 1000))
 
         n_fft = 400

